deepset_cloud_sdk/_api/deepset_cloud_api.py

```diff
@@ -67,6 +68,12 @@ async def factory(cls, config: CommonConfig) -> AsyncGenerator[DeepsetCloudAPI,
         async with httpx.AsyncClient() as client:
             yield cls(config, client)
 
+    @retry(
```

Comment:
do we only want this on the get? Is it worth putting it on the other methods too?